.prettierrc

```
@@ -0,0 +1 @@
+{}
```

Here is the default `.prettierrc` configuration that we use in other projects.

```suggestion
{
	"printWidth": 110,
	"singleQuote": true,
	"semi": true,
	"useTabs": true
}

```